plugins/clopen.py

```diff
@@ -215,6 +215,8 @@ class Channel:
     extension: Mapped[int] = mapped_column(BigInteger, nullable=False)
     # When to transition to the respective next state
     expiry: Mapped[Optional[datetime]] = mapped_column(TIMESTAMP)
+    creatation_date: Mapped[Optional[datetime]] = mapped_column(TIMESTAMP)
+
```

Comment:
It seems you didn't run Black. Please refer to [CONTRIBUTING.md](https://github.com/discord-math/bot/blob/main/CONTRIBUTING.md) if you want to contribute to the repo.